go.mod

```diff
@@ -25,7 +26,6 @@ require (
 	k8s.io/client-go v0.19.2
 	k8s.io/code-generator v0.19.2
 	k8s.io/component-base v0.19.2
-	k8s.io/klog/v2 v2.3.0
```

Comment:
isn't go mod cool?

Reply:
yeah, I noticed this change, too. It's... eh.